auto_sizing/targets.py

```diff
@@ -73,10 +76,12 @@ def from_file(self, target_dict: Dict, path: str) -> List[Segment]:
 
         return Segment_list
 
-    def _make_desktop_targets(self, target: Dict[str, str]) -> List[Segment]:
+    def _make_desktop_targets(
+        self, target: Dict[str, str], start_date: Optional[str] = None
```

Comment:
Why is this parameter called `target` here but `recipe` in `_desktop_sql`?

Reply:
+1 -- If these are the same thing, naming should be consistent. I think there are a few other places where "target" and "recipe" are used alternately/interchangeably.